skimage/transform/tests/test_warps.py

```diff
@@ -517,3 +520,78 @@ def test_zero_image_size():
     with testing.raises(ValueError):
         warp(np.zeros((10, 10, 0)),
              SimilarityTransform())
+
+
+def test_linear_polar_mapping():
+    output_coords = np.array([[0, 0],
+                             [0, 90],
+                             [0, 180],
+                             [0, 270],
+                             [99, 0],
+                             [99, 180],
+                             [99, 270],
+                             [99, 45]])
+    ground_truth = np.array([[100, 100],
+                             [100, 100],
+                             [100, 100],
+                             [100, 100],
+                             [199, 100],
+                             [1, 100],
+                             [100, 1],
+                             [170.00357134, 170.00357134]])
+    k_angle = 360 / (2 * np.pi)
+    k_radius = 1
+    center = (100, 100)
+    coords = _linear_polar_mapping(output_coords, k_angle, k_radius, center)
+    assert np.allclose(coords, ground_truth)
+
+
+def test_log_polar_mapping():
+    output_coords = np.array([[0, 0],
+                              [0, 90],
+                              [0, 180],
+                              [0, 270],
+                              [99, 0],
+                              [99, 180],
+                              [99, 270],
+                              [99, 45]])
+    ground_truth = np.array([[101, 100],
+                             [100, 101],
+                             [99, 100],
+                             [100, 99],
+                             [195.4992586, 100],
+                             [4.5007414, 100],
+                             [100, 4.5007414],
+                             [167.52817336, 167.52817336]])
+    k_angle = 360 / (2 * np.pi)
+    k_radius = 100 / np.log(100)
+    center = (100, 100)
+    coords = _log_polar_mapping(output_coords, k_angle, k_radius, center)
+    assert np.allclose(coords, ground_truth)
+
+
+def test_linear_warp_polar():
+    radii = [5, 10, 15, 20]
+    image = np.zeros([51, 51])
+    for rad in radii:
+        rr, cc, val = circle_perimeter_aa(25, 25, rad)
+        image[rr, cc] = val
+    warped = warp_polar(image, radius=25)
+    profile = warped.mean(axis=0)
+    peaks = peak_local_max(profile)
+    assert np.alltrue([peak in radii for peak in peaks])
+
+
+def test_log_warp_polar():
```

Comment:
We need at least one more test, `test_invalid_scaling`, using `with testing.raises(ValueError):`.